### PR TITLE
Cleanup + Build system

### PR DESCRIPTION
**Pull request by dunkelstern**

### CODE REVIEW

File: package/atpkg/atpkg_package.py [hunk 1]
```diff
@@ -60,3 +62,11 @@ def parse(self, key, value):
...
 		else:
 			raise PackageError("Unknown item '{name}'".format(name=key))
+
+	def task_for_file(self, filename):
+		sourcePath = os.path.relpath(filename, start=os.path.dirname(self.root_path))
+		for task in self.tasks.values():
+			if not isinstance(task, LLBuildTask): continue
+			if sourcePath in task.source_files:
+				return task
+		return None
```
Comment (drewcrawford) [inline package/atpkg/atpkg_package.py:72]: Ideally, we should log a warning if multiple tasks match.

File: package/clojure/parsetab.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
...
...,25,26,],[9,9,9,-8,-6,-7,9,-9,-10,-1,-3,-2,9,-5,-11,-18,9,-4,-16,9,-14,9,-12,-17,-15,-13,]),'RPAREN':([3,4,5,7,8,9,11,12,13,14,15,16,18,19,21,22,23,24,25,26,],[-8,-6,-7,-9,-10,-1,-3,-2,21,-5,-11,-18,-4,-16,-14,26,-12,-17,-15,-13,]),'QUOTED':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[11,11,11,-8,-6,-7,11,-9,-10,-1,-3,-2,11,-5,-11,-18,11,-4,-16,11,-14,11,-12,-17,-15,-13,]),'RBRACE':([1,3,4,5,7,8,9,11,12,14,15,16,17,18,19,21,23,24,25,26,],[16,-8,-6,-7,-9,-10,-1,-3,-2,-5,-11,-18,24,-4,-16,-14,-12,-17,-15,-13,]),'ATOM':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[12,12,12,-8,-6,-7,12,-9,-10,-1,-3,-2,12,-5,-11,-18,12,-4,-16,12,-14,12,-12,-17,-15,-13,]),'LPAREN':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[13,13,13,-8,-6,-7,13,-9,-10,-1,-3,-2,13,-5,-11,-18,13,-4,-16,13,-14,13,-12,-17,-15,-13,]),'KEYWORD':([0,1,2,3,4,5,6,7,8,9,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,],[14,14,14,-8,-6,-7,14,-9,-10,-1,-3,-2,14,-5,-11,-18,14,-4,-16,14,-14,14,-12,-17,-15,-13,]),'RBRACKET':([3,4,5,6,7,8,9,11,12,14,15,16,18,19,20,21,23,24,25,26,],[-8,-6,-7,19,-9,-10,-1,-3,-2,-5,-11,-18,-4,-16,25,-14,-12,-17,-15,-13,]),}
```
Comment (drewcrawford) [inline package/clojure/parsetab.py:10]: Is this file autogenerated?  I find it in my commits from time to time, not sure if ok to remove

**Comment by drewcrawford** [comment]
Can't test this (#13) but this doesn't break anything I'm working on.  Let's take it.

File: package/clojure/parsetab.py (hunk 2 above)
Comment (dunkelstern) [inline package/clojure/parsetab.py:10]: Generated by ply, not sure when or how it is generated and why it changes all the time

**Pull request merged**